src/core/js/libraries/inview.js

```diff
@@ -392,6 +420,7 @@
             var body = $("body")[0].getBoundingClientRect();
             //make sure to get height and width independently if getBoundingClientRect doesn't return height and width;
             measurements.supplimentDimensions = (body.width === undefined);
+            loop.hasRaf = (window.requestAnimationFrame && window.cancelAnimationFrame);
```

Comment:
is it compatible with all browsers ex. Mozilla has window.mozRequestAnimationFrame.